docs/overview.rst

```diff
@@ -238,55 +237,53 @@ You have two choices:
+
+  - one can implement custom types.
 
-  - it is possible to write extensions which expose module-level functions,
-    with all the various kinds of calling conventions
+  - there is support for raising and catching exceptions.
 
-  - there is a limited support for argument parsing (only a couple of basic
-    types actually work)
+  - debug mode has been implemented and can be activated at run-time without
+    recompiling. It can detect leaked handles or handles used after
+    being closed.
 
-  - there is support for raising and catching exceptions
+  - wheels can be build for HPy extensions with `python setup.py bdist_wheel`
+    and can be installed with `pip install`.
 
   - it is possible to choose between the :term:`CPython ABI` and the
-    :term:`HPy Universal ABI` when compiling an extension module
+    :term:`HPy Universal ABI` when compiling an extension module.
 
   - extensions compiled with the CPython ABI work out of the box on
-    CPython
+    CPython.
 
   - it is possible to load HPy Universal extensions on CPython, thanks to the
-    ``hpy.universal`` package
+    ``hpy.universal`` package.
 
   - it is possible to load HPy Universal extensions on
-    PyPy (using the PyPy `hpy branch <https://foss.heptapod.net/pypy/pypy/tree/branch/hpy>`_)
+    PyPy (using the PyPy `hpy branch <https://foss.heptapod.net/pypy/pypy/tree/branch/hpy>`_).
 
   - it is possible to load HPy Universal extensions on `GraalPython
-    <https://github.com/graalvm/graalpython>`_
+    <https://github.com/graalvm/graalpython>`_.
 
 
 However, there is still a long road before HPy is usable for the general
 public. In particular, the following features are on our roadmap but have not
 been implemented yet:
 
-  - it is not possible to write custom types (like NumPy's ndarray)
-    in C. There is already a WIP branch to address this issue
-
-  - only a handful of the original Python/C functions have been ported to
-    HPy. Porting most of them is straighforward, so for now the priority is to
-    work on the "hard" features to prove that the HPy approach works, and we
-    will port new functions as needed
-
-  - the debug mode simply does not exist (yet!)
-
-  - there is no standard/easy way to integrate HPy into a
-    ``distutils/setuptools`` based workflow. The only way to compile HPy
-    extensions right now is to manually clone the git repo and tweak your
-    ``setup.py``. Eventually, we plan to offer a workflow which integrates
-    seamlessly with ``pip``, ``setuptools``, etc.
+  - many of the original Python/C functions have not been ported to
+    HPy yet. Porting most of them is straighforward, so for now the priority
+    is to work on the "hard" features to prove that the HPy approach works, and
+    we will port new functions as needed
 
   - there is no integration with Cython. The medium-term plan is to extend
-    Cython to automatically generate HPy-compatible C code
+    Cython to automatically generate HPy-compatible C code.
 
 
```

Comment:
more future work:
- `HPyField`/`HPy_Load`/`HPy_Store`
- improving the debug mode so that it also tells you the C traceback of the place who leaked a handle or used a closed one
- design a way to define and access a C-level module state

Reply:
Added.